### DISCUSSION TITLE
How to tell what measurement units are ?

### DISCUSSION THREAD

**Discussion by macosta541** [mtex-toolbox/mtex#858] [Ask Anything] [answered]
Hello,

Please excuse the naivete of this question, but how do I tell what the "measurement units" of my dataset are?

For example, I load the ebsd data, which has scan units of microns (is scan unit the same thing as measurement unit?). I then use calcGrains to calculate the grains and I want to know what the distribution of grain sizes is. I am not sure what the best practice is, but figure I can use hist(grains) to get the grain size distribution  - which looks like it is in pixels? can I convert that to microns?  The documentation says that the output is in "measurement units".

The other thing I did was to calculate the grain diameters, and then to plot those as a histogram as well. This looks like it  is in microns. (?)

Many thanks,
Marisa

![image](https://user-images.githubusercontent.com/74112001/111466600-ca259d80-8723-11eb-9ff5-b22b70b74b7c.png)

**Comment by kilir** [comment, marked as answer]
Hi,
`grains.equivalentRadius`, `grains.area` etc are all in measurement units, microns in most cases, `grains.grainSize` is in Pixels. Weighting distributiosn by are (what grain2d/histogram does) is probably a much better description than a frequency distribution.
The measurement units are the units of your x,y coordinates.

Hope this helps.

Cheers,
Rüdiger

**Reply by macosta541** [reply]
Very helpful, thank you!

Best,
Marisa
